packages/service/scripts/prod-init.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
any reason not to keep these shebangs consistent?

Reply:
not really ... this was just copy pasta from another repo, and nothing in this script requires "new bash" -- the `env bash` pattern is for poor folks stuck on macOS that is constrained to old versions of bash before it went GPL3. But this script only barely even needs bash -- if we got rid of `set -o pipefail` and just kept `set -xeu` it'd work with `/bin/dash` which is about the most limited posix sh you can find :)

And for just doing `exec node . ...` we don't even need the `set ...` calls, those are just there for cases where we want to add fancier stuff in specific services...

Anyways, rambling aside, env bash is safer for adding features in real services

Reply:
alrighty!